src/lib/phase.ml

```diff
@@ -0,0 +1,95 @@
+
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Type definitions *)
+(* ************************************************************************* *)
+
+type id = Id.t [@@deriving yojson]
+
+type t = {
+  id : id;
+  name : string;
+  competition : Competition.id;
+  phase_order : string;
+  judge_artefact : string;
+  head_judge_artefact : string;
```

Comment:
These two fields should contain a `Artefact.Descr.t` and not a string, and it'd be better if their names had a `_descr` added at the end.